src/wallet.js

```diff
@@ -351,6 +351,8 @@ MyWallet.initializeWallet = function (pw, decryptSuccess, buildHdSuccess) {
     var loadExternalFailed = function (message) {
       console.warn('wallet.external not set:', message);
     };
+    // try save guid.
+    MyWallet.wallet.saveGUIDtoMetadata();
```

Comment:
yes, to check if the guid is already saved is already a call and I discussed with matt to store a boolean on the json to know if it is already saved and he did not wanted to pollute more the json.

By the way. I will move this changes to stats branch and make it all in once to avoid conflicts